packages/jellyfish-api-core/src/category/masternode.ts

```diff
@@ -198,6 +198,17 @@ export class Masternode {
     return await this.client.call('listgovs', [], 'bignumber')
   }
 
+  /**
+   * Checks that custom transaction was affected on chain
+   *
+   * @param {string} transactionId transaction hash
+   * @param {number} blockHeight height of block which contain transaction
+   * @return {Promise<boolean>} indicate that custom transaction was affected on chain
+   */
+  async isAppliedCustomTransaction (transactionId: string, blockHeight: number): Promise<boolean> {
+    return await this.client.call('isappliedcustomtx', [transactionId, blockHeight], 'bignumber')
```

Comment:
does not need `bignumber` precision.
```suggestion
    return await this.client.call('isappliedcustomtx', [transactionId, blockHeight], 'number')
```